### ISSUE TITLE
Convert to addon v2 format

### ISSUE THREAD

**Issue by sandydoo** [sandydoo/ember-google-maps#155]

**Comment by acorncom** [comment]
@sandydoo from various comments I've seen you post, looks like your involvement with Ember may be lessening these days. Would you be interested in someone picking up this issue / helping maintain this addon long-term? I'm using it for various client projects and continue to find it quite useful, so might be willing to help with maintenance ...

**Comment by sandydoo** [comment]
Yeah, of course! If the v2 API is stable enough, go for it 🙂 I'm always available to review and help guide PRs.
